asciinema/schema_evolution.py

```diff
@@ -0,0 +1,53 @@
+from generate_asciinema_cast import Cast
+
+sequence = [
+    (
+        "ALTER TABLE taxis ADD COLUMN fare_per_distance_unit float AFTER trip_distance;",
+        "Time taken: 0.671 seconds",
+    ),
+    (
+      "DESCRIBE TABLE nyc.taxis;",
+      """VendorID    string
+tpep_pickup_datetime    string
```

Comment:
indentation seems off here?

Reply:
I think this is to avoid indentation in the cast